src/Block/Adminhtml/Product/Edit/CreateProjectButton.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EasyTranslate\Connector\Block\Adminhtml\Product\Edit;
+
+use Magento\Framework\App\RequestInterface;
+use Magento\Framework\UrlInterface;
+use Magento\Framework\View\Element\UiComponent\Control\ButtonProviderInterface;
+
+class CreateProjectButton implements ButtonProviderInterface
+{
+    /**
+     * @var UrlInterface
+     */
+    private $url;
+
+    /**
+     * @var RequestInterface
+     */
+    private $request;
+
+    public function __construct(
+        UrlInterface $url,
+        RequestInterface $request
+    ) {
+        $this->url     = $url;
+        $this->request = $request;
+    }
+
+    public function getButtonData(): array
+    {
+        return [
+            'id'         => 'easytranslate_create_project',
+            'label'      => __('Create Easytranslate Project'),
+            'on_click'   => sprintf(
+                "location.href = '%s';",
```

Comment:
Can we open it in a new tab / window? I think we should either do this or have a confirmation message that all unsaved changes are lost. I think a new tab / window would be nicer, though, because it is less obtrusive.